src/Glue.elm

```diff
@@ -299,17 +242,17 @@ subscriptions (Glue { msg, get }) subscriptions_ mainSubscriptions =
             ]
 
 
-{-| Subscribe to subscriptions when model is in some state.
+{-| Subscribe to child's `subscriptions` based on some condition in the parent module.
 
     type alias Model =
-        { subModuleSubsOn : Bool
-        , subModuleModel : SubModule.Model
+        { useCounter : Bool
+        , counterModel : Counter.Model
```

Comment:
The rest of the docs uses either Counter or Child when talking about the submodule. So I'd keep it consistent...